torch/testing/_internal/common_methods_invocations.py

```diff
@@ -740,7 +740,10 @@ def to_float(start, end, step):
     for start, end, step in samples:
         if start is None:
             assert step is None
+            # Pass end as positional arg
             yield SampleInput(end, kwargs={"dtype": dtype, "device": device})
+            # (Similar to) calling torch.arange(end=3)
+            yield SampleInput(0, kwargs={"end": end, "dtype": dtype, "device": device})
```

Comment:
This one we cannot test properly really. This is a limitation of OpInfos.

Reply:
A limitation of how we express SampleInputs, you mean? Because we assume the SampleInput has at least one positional argument?

Reply:
exactly

Reply:
In retrospect, I think it wasn't a good design to single out the first arg to `SampleInput`. Taking the positional args as `*args` and the rest kw-args may have done the trick and would be more extensible.